i18n/en.json

```diff
@@ -4271,6 +4271,10 @@
     "id": "api.user.view_archived_channels.get_users_in_channel.app_error",
     "translation": "Cannot retrieve users for an archived channel"
   },
+  {
+    "id": "api.users.invalid_emails.enable_open_server.app_error",
+    "translation": " "
```

Comment:
do we maybe need some input from the docs team here?

Reply:
No, we are trying to avoid adding error messages to the server. You might need some input for the frontend error from this ID